src/serializers/type_serializers/float.rs

```diff
@@ -85,16 +103,11 @@ impl TypeSerializer for FloatSerializer {
         serializer: S,
         include: Option<&Bound<'_, PyAny>>,
         exclude: Option<&Bound<'_, PyAny>>,
+        // TODO: Merge extra.config into self.inf_nan_mode?
```

Comment:
Yep good observation, though I guess it's a follow-up which might change behaviour.